site/docs/v1/tech/email-templates/email-templates.adoc

```diff
@@ -459,7 +489,9 @@ The User object, see the User API for field definitions of a User.
 [field]#verificationId# [type]#[String]#::
 The verification Id intended to be used by the link:/docs/v1/tech/apis/registrations#verify-a-user-registration[Verify a User Registration] API.
 
-
+[field]#verificationOneTimeCode# [type]#[String]#::
+// TODO Add link to additional documentation as we complete
+The verification One Time Code (OTP) to be used with the Gated Registration workflow. The user enters this code to open the 'gate' and allow the application registration to continue.
```

Comment:
Not sure that the OTP acronym is helpful here (and I'm a bit confused, wouldn't it be 'OTC'? ).

"The user enters this code to open the 'gate' and allow the email verification to continue." =>
"The user enters this code to verify their email."